### PR TITLE
Sprout update

### PR DESCRIPTION
**Pull request by mfalesni**
* RHEV-M support
* Celery beat for scheduling periodic tasks
* This enabled the update script to be actually working (probably)

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8cee6fa22e116e8cec42d0506d1726be0c77828a:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./utils/mgmt_system.py:1158:23: E226 missing whitespace around arithmetic operator
./utils/mgmt_system.py:1162:23: E226 missing whitespace around arithmetic operator
```
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a0f4ecfbc4c618639f7985ad43003f49a995f791:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d9651301a89f45b1284a84134bece01245e6142d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/mgmt_system.py [hunk 1]
```diff
@@ -1144,6 +1145,27 @@ def deploy_template(self, template, *args, **kwargs):
     def remove_host_from_cluster(self, hostname):
         raise NotImplementedError('remove_host_from_cluster not implemented')
 
+    def mark_as_template(self, vm_name):
```
Comment (seandst) [inline utils/mgmt_system.py]: The innocent-sounding name doesn't really line up with "Take a VM, stop it, turn it into a template, and delete it." This should be solved with a docstring explaining what this does and why it's needed.

Comment (mfalesni) [inline utils/mgmt_system.py]: Yup. I mimic the VMware's behaviour here.

**Comment by dajoRH** [comment]
Lint report for commit 882e9cf1ca3db1164447f1371cc652fcceb63bb0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by seandst** [comment]
:+1:

**Pull request merged**